### ISSUE TITLE
pq: password authentication failed  (fresh install)

### ISSUE THREAD

**Issue by serofir** [bakape/meguca#430]
After install, I tried turn on meguca by  ./meguca -a :80 .
And it says, pq: password authentication failed for user "meguca" .
I used this https://github.com/bakape/meguca/blob/master/docs/installation.md guide to install.

**Comment by bakape** [comment]
If you created the PostgreSQL user "meguca" with a password other than
"meguca", you should pass that in the PostgreSQL connection arguments flag
(`-d`).

**Comment by serofir** [comment]
I tried to remove the user meguca from postgresql, with (dropuser -i  meguca).
But this does not work, I can not create a new user meguca, because of error "createuser: creation of new role failed: ERROR:  role "meguca" already exists"
Then I tried to remove ROLE, I used the psql client, and command "DROP ROLE meguca;", and it says error again "ERROR:  role "meguca" cannot be dropped because some objects depend on it
DETAIL:  owner of database meguca".
After all this, I removed Postrgres by "apt-get remove", and again install it.
I tried to add user meguca, and postgres says error : "createuser: creation of new role failed: ERROR:  role "meguca" already exists"
